modules/generic_worker/templates/run-generic-worker.sh.erb

```diff
@@ -14,6 +14,39 @@ rm -f tasks-resolved-count.txt
 # wrapper script...
 /usr/local/bin/generic-worker "$@" 2>&1 | logger -t generic-worker -s
 
+exitstatus=( ${PIPESTATUS[0]} )
+
+echo $exitstatus
+
+export TASKCLUSTER_CLIENT_ID="<%= @quarantine_client_id %>"
+export TASKCLUSTER_ACCESS_TOKEN="<%= @quarantine_access_token %>"
+
+if [ $exitstatus -eq 69 ]; then
+    if [ -f /Users/cltbld/quarantined ]; then
+        echo "Worker is quarantined"
+    else
+        echo "Quarantine worker"
+        # quarantine the worker
```

Comment:
I prefer: `# quarantine the worker using github.com/mozilla-platform-ops/quarantine-worker CLI`.

The quarantine-worker project also needs a `README.md` to explain command line args and explain what it does, and why it is useful etc.

Reply:
+1